### PR TITLE
use bangladesh incidence rate raster for targeting

### PR DESCRIPTION
**Pull request by christinaalam**

### CODE REVIEW

File: packages/ocvImpact/R/utils_targeting.R [hunk 1]
```diff
@@ -55,6 +55,9 @@ load_targets_by_country <- function(datapath, modelpath, country){
       message(paste0("Loading ", datapath, "/incidence/afro_2010-2016_lambda_5k_mean.tif"))
       afr <- raster::raster(paste0(datapath, "/incidence/afro_2010-2016_lambda_5k_mean.tif"))
     }
+    
+    bgd <- raster::raster(paste0(datapath, "/incidence/BGD_incid_5k_100.tif"))
```
Comment (eclee25) [inline packages/ocvImpact/R/utils_targeting.R:59]: I'm a bit confused here... are we loading the bgd raster even if the modeled country is not bgd? If the modeled country is in africa, we only need to load the afr raster.

Comment (christinaalam) [reply packages/ocvImpact/R/utils_targeting.R:59]: Good point, I'll put this inside an if statement

File: packages/ocvImpact/R/utils_targeting.R [hunk 2]
```diff
@@ -55,15 +55,34 @@ load_targets_by_country <- function(datapath, modelpath, country){
...
-    ## summarize rasters to admin level
-    incid2 <- exactextractr::exact_extract(afr, shp, 'mean') ## could add population weight here for better incidence estimate but need to project population to the incidence grid
+    ## summarize rasters to admin level (BGD, non-raster, and african raster countries)
+    if (country == "BGD"){
+      bgd <- raster::raster(paste0(datapath, "/incidence/BGD_incid_5k_100.tif"))
+      incid2 <- exactextractr::exact_extract(bgd, shp, 'mean') ## to use BGD incidence raster for targeting
+    } else if (country %in% c("AFG", "HTI", "IRN", "IRQ", "NPL", "PAK", "PHL", "THA", "YEM", "IND")) { #for non-raster countries
+       number_samples <- config$incid$num_samples
+       file_path <- paste0(datapath, "/incidence/",country, "_incid_5k_", number_samples, ".tif")
+       rm(number_samples) ##to remove global variable
```
Comment (eclee25) [inline packages/ocvImpact/R/utils_targeting.R:74]: I don't understand why this is being removed shortly after being assigned...

Comment (christinaalam) [reply packages/ocvImpact/R/utils_targeting.R:74]: I added the number_samples global variable here to make sure we can use the number of samples from the config to get the name of the generated incidence rate raster for that country and I removed it to make sure we're not keeping unneeded global variables. I can keep it if you don't mind having global variables

**Pull request merged**